e2e-jar-test/standard/src/test/java/net/snowflake/StandardIngestE2ETest.java

```diff
@@ -28,7 +28,6 @@ public void basicTest() throws InterruptedException {
   }
 
   @Test
-  @Ignore("Takes too long to run")
```

Comment:
I will remove in next commit before pushing!

Reply:
There is an equivalent long running test for FIPS JDBC driver in [FipsIngestE2ETest.java](https://github.com/snowflakedb/snowflake-ingest-java/blob/master/e2e-jar-test/fips/src/test/java/net/snowflake/FipsIngestE2ETest.java#L36), could you run it, as well?

Reply:
Will do!